performance/performance_tuning.rst

```diff
@@ -35,7 +34,62 @@ PostgreSQL tuning
 
 The Citus master partitions an incoming query into fragment queries, and sends them to the workers for parallel processing. The workers are just extended PostgreSQL servers and they apply PostgreSQL's standard planning and execution logic for these queries. So, the first step in tuning Citus is tuning the PostgreSQL configuration parameters on the workers for high performance.
 
-This step involves loading a small portion of the data into regular PostgreSQL tables on the workers. These tables would be representative of shards on which your queries would run once the full data has been distributed. Then, you can run the fragment queries on the individual shards and use standard PostgreSQL tuning to optimize query performance.
+While adjusting parameters to tune the workers we want fast iterations. To begin the tuning process, create a Citus cluster and load a small portion of your data in it. This will keep operations fast. Once the data is loaded, use the EXPLAIN command on the master node to inspect performance. 
```

Comment:
This paragraph confuses me a bit. Especially these two sentences:
"While adjusting parameters to tune the workers we want fast iterations"
and
"This will keep operations fast". 

What do we want to communicate here with those statements?